charts/monasca/templates/client-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.client.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "client.fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Values.client.name }}-deployment"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        component: "{{ .Values.client.name }}-deployment"
```

Comment:
you can take out deployment the only reason we did that for the agent is we need to differentiate from different instances of it